modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -61,16 +62,16 @@ NOTE: Clicking each section of the message flow expands it. A green checkmark ne
 
 . For a new message type, depending on the message standard, refer to the following:
 +
-* xref:document-types.adoc[Message (Document) Types]
+* xref:document-types.adoc[Message Types]
 +
-. (Optional) Click the *View* link to edit the validation settings and then click the hostname. For more information, see xref:x12-receive-read-settings.adoc[X12 Settings - Receive from Partner].
-
-. (Optional for X12 partner requirements) Configure the type of functional acknowledgment, and the endpoint to which you send it. Refer to the following information for endpoint information:
+. (Optional) Click the *View* link to edit the validation settings and then click the hostname.
+For more information, see xref:x12-receive-read-settings.adoc[X12 Settings - Receive from Partner].
 
+. (Optional for X12 partner requirements) Configure the type of functional acknowledgment, and the endpoint to which you send it. Refer to the following for endpoint information:
 +
-* <<HTTP(S) Protocol Send Settings>>
+* xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
 
-* <<SFTP Protocol Send Settings>>
+* xref:endpoint-sftp-receive-target.adoc[SFTP Receive Endpoint Settings]
 +
 . Click the *Map* section and then click *Import* to choose a DataWeave map.
```

Comment:
This could represent the beginning of a Create a Map task within Create a Message Flow.
You could add a subhead, and an intro something like...:

"After you finish creating the source of your message flow, you continue working in the *New message flows* link within your sandbox or production Anypoint Partner Manager environment to create the map portion of your message flow."

Reply:
Yes, let's look at that for next release.